src/Simulation/Simulators/QuantumProcessor/Gate.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Simulation.QuantumProcessor
+{
+    public partial class QuantumProcessorDispatcher
+    {
+
+        public class QuantumProcessorApplyGate : Extensions.Gate
```

Comment:
As a public type, this should have an API documentation comment, even if that comment is just an inheritdoc.

Reply:
Sure.

---
In reply to: [410524961](https://github.com/microsoft/qsharp-runtime/pull/172#discussion_r410524961) [](ancestors = 410524961)